examples/simple/server.py

```diff
@@ -57,14 +57,14 @@ def dropClient(cli, errors=None):
 
 while 1:
     try:
-        r,w,_ = select.select([server]+clients.keys(), writers.keys(), [])
+        r,w,_ = select.select([server]+list(clients.keys()), list(writers.keys()), [])
```

Comment:
lists removed(4852ef4) and then re-added(b2ff5be) as will not work under python3.  Spacing fixed in those commits.